.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,5 +1,5 @@
 blank_issues_enabled: true
 contact_links:
   - name: GMT Community Forum
-    url: https://forum.generic-mapping-tools.org/
+    url: https://forum.generic-mapping-tools.org/c/questions/pygmt-q-a/11
```

Looks like
```suggestion
    url: https://forum.generic-mapping-tools.org/c/questions/pygmt-q-a
```
redirects. Not sure what the "11" at the end of the URL is for, may be good to remove it?

---

Seems OK to remove it.